typescript/phoenix-sdk/tests/swap.ts

```diff
@@ -23,16 +24,19 @@ export async function swap() {
   );
 
   const marketAddress = new PublicKey(
-    "5iLqmcg8vifdnnw6wEpVtQxFE4Few5uiceDWzi3jvzH8"
+    "3MZskhKUdNRkeMQ6zyNVSJcCx38o79ohwmSgZ2d5a4cu"
   );
   const marketAccount = await connection.getAccountInfo(marketAddress);
   if (!marketAccount)
     throw Error(
       "Market account not found for address: " + marketAddress.toBase58()
     );
-  const marketData = Phoenix.deserializeMarketData(
-    Buffer.from(marketAccount.data)
-  );
+
+  let market = await Phoenix.Market.load({
+    connection,
+    address: marketAddress,
+  });
```

Comment:
can use const here

Reply:
do you have a preferred linter / code-checker we can use to detect these on pre-commit?

Reply:
yeah i can make a note for myself to add that